README.md

```diff
@@ -70,25 +70,31 @@ Initialize a project:
 ```sh
 $ mkdir pipcook-example && cd pipcook-example
 $ pipcook init
-$ cd pipcook-project
 ```
 
+### Show Cases
```

Comment:
How about "Playground" or "Pipboard"?

Reply:
I guess playground is more attractive. I would use 'Playground' :)